lib/pagseguro/errors.rb

```diff
@@ -20,17 +20,35 @@ def add(response)
 
     private
     def process(response)
-      @messages << error_message(:unauthorized, "Unauthorized") if response.unauthorized?
-      @messages << error_message(:forbidden, "Forbidden") if response.forbidden?
-      @messages << error_message(:not_found, "Not found") if response.not_found?
+      @messages << error_message(
+        error_to_sym(response.error),
+        error_to_human(response.error)
+      ) if response.error?
 
       response.data.css("errors > error").each do |error|
         @messages << error_message(error.css("code").text, error.css("message").text)
-      end if response.bad_request?
+      end if response.error == Aitch::BadRequestError
     end
```

Comment:
Yes, this is a good idea use:
```
def process(response)
  return unless response.error?
  @messages << error_message(
    error_to_sym(response.error),
    error_to_human(response.error)
  )
end
```
But related about:

  response.data.css("errors > error").each do |error|
    @messages << error_message(error.css("code").text, error.css("message").text)
  end if response.error == Aitch::BadRequestError